sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -996,6 +1001,25 @@ mod checked {
         builder
     }
 
+    fn setup_bridge_create(
+        mut builder: ProgrammableTransactionBuilder,
+    ) -> ProgrammableTransactionBuilder {
+        let bridge_uid = builder
+            .input(CallArg::Pure(UID::new(SUI_BRIDGE_OBJECT_ID).to_bcs_bytes()))
+            .unwrap();
+        // Set chain id to sui devnet
+        // TODO: set this according to ChainIdentifier
```

Comment:
when is this `TODO` going to be done?

Reply:
this is done in https://github.com/MystenLabs/sui/pull/16242/